adhocracy4/ratings/static/ratings/react_ratings.jsx

```diff
@@ -65,10 +65,6 @@ class RatingBox extends React.Component {
       }.bind(this))
   }
 
-  updateUserRating (data) {
-    this.state.ratings[this.state.userRatingIndex] = data
-  }
-
```

Comment:
function never called so deleted?

Reply:
@fuzzylogic2000 tested with linking to Opin and seems to work but could do with some hand holding, do you think this is ok?

Reply:
Nice! I think it's a good catch and will also try it in OPIN and maybe also mB.